chain/index/README.MD

```diff
@@ -107,6 +107,8 @@ type IndexValidation struct {
 	IndexedMessagesCount uint64
 	// IndexedEventsCount signifies the number of indexed events for the canonical tipset at this epoch.
 	IndexedEventsCount uint64
+	// IndexedEventEntriesCount signifies the number of indexed event entries for the canonical tipset at this epoch.
```

Comment:
_"is the number of"_

Please can I request you to fix it in the above as well ?

Reply:
Done